hadoop-ozone/ozone-manager/src/test/java/org/apache/hadoop/ozone/om/TestOmMetadataManager.java

```diff
@@ -698,6 +698,7 @@ private void testGetExpiredOpenKeysExcludeMPUKeys(
               RatisReplicationConfig.getInstance(ONE), new OmKeyLocationInfoGroup(0L, new ArrayList<>(), true))
           .setCreationTime(expiredOpenKeyCreationTime)
           .build();
+      assertTrue(keyInfo.getModificationTime() > 0L);
```

Comment:
Please use `assertThat` for better failure message (see HDDS-9951):

```suggestion
      assertThat(keyInfo.getModificationTime()).isPositive();
```